base/linalg/arnoldi.jl

```diff
@@ -363,6 +364,35 @@ iterations derived from [`eigs`](@ref).
 # Example
 
 ```jldoctest
+julia> X = sprand(3, 3, 0.5);
```

Comment:
Can not use `sprand` in a doctest. Also, small floating numbers like this might not be the same all the time, see https://github.com/JuliaLang/julia/pull/20493 where this doctests originates from.